src/schemas/json/substreams-manifest-schema.json

```
@@ -0,0 +1,304 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "additionalProperties": false,
+  "description": "A Schema to support in making Substreams manifest files",
+  "name": "Substreams Manifest Schema",
+  "properties": {
+    "specVersion": {
+      "enum": ["v0.1.0"]
+    },
+    "package": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
```

```suggestion
          "description": "A package name\nhttps://substreams.streamingfast.io/reference-and-specs/manifests#package"
          "type": "string",
```

Required fix (multiple violations): [add key explanation](https://github.com/SchemaStore/schemastore/blob/master/CONTRIBUTING.md#best-practices).